flink-java/src/main/java/org/apache/flink/api/java/io/TextOutputFormat.java

```diff
@@ -18,65 +18,73 @@
 
 package org.apache.flink.api.java.io;
 
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.api.common.io.FileOutputFormat;
+import org.apache.flink.core.fs.Path;
+
 import java.io.IOException;
 import java.io.Serializable;
 import java.nio.charset.Charset;
 import java.nio.charset.IllegalCharsetNameException;
 import java.nio.charset.UnsupportedCharsetException;
 
-import org.apache.flink.annotation.PublicEvolving;
-import org.apache.flink.api.common.io.FileOutputFormat;
-import org.apache.flink.core.fs.Path;
-
+/**
+ * {@link FileOutputFormat} that stores values by calling {@link Object#toString()} method.
```

Comment:
This isn't necessarily correct, as a `TextFormatter` might be used.

How about
```
A {@link FileOutputFormat} that writes objects to a text file.

<p>Objects are converted to Strings using either {@link Object#toString()} or a {@link TextFormatter}.
```